lib/Graph/Nodes.cpp

```diff
@@ -248,7 +248,11 @@ static void verifyArithmetic(NodeValue LHS, NodeValue RHS, NodeValue res) {
 }
 
 static void verifyRelu(NodeValue src, NodeValue dest) {
-  checkSameType(src, dest);
+  if (src.getType()->isQuantizedType()) {
+    checkSameShape(src, dest);
```

Comment:
Maybe add a check here that the output quantization params min must be non-negative?